packages/suite/src/utils/wallet/transactionUtils.ts

```diff
@@ -679,3 +679,8 @@ export const getBlockExplorerUrl = (tx: WalletAccountTransaction) => {
     const network = getNetwork(tx.symbol);
     return `${network!.explorer.tx}${tx.txid}`;
 };
+
+export const isTxFinal = (tx: WalletAccountTransaction, confirmations: number) => {
+    // checks RBF status
+    return !!(!tx.rbf || confirmations > 0);
```

Comment:
Is it necessary to cast it into a boolean here? Both, `!tx.rbf` and `confirmations > 0`, should return bools already. And I guess for such a short function, the return statement can be omitted as well.

Reply:
that what eslint did by itself :D I can't place the comment inside if I omit the return statement 🤔

Reply:
ok, removed !!, looks like now eslint is fine with it 🤷‍♂️